### PR TITLE
Red Dead Redemption 2 Provider

### PR DESCRIPTION
**Pull request by Majid-J**
Added a provider for Red Dead Redemption 2.

### CODE REVIEW

**Comment by kingthorin** [comment]
Could you remove the unrelated commits please

File: src/main/java/net/datafaker/providers/videogame/RedDeadRedemption2.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package net.datafaker.providers.videogame;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+/**
+ * Red Dead Redemption 2 is an action-adventure game developed and published by Rockstar Games.
+ *
+ * @since 1.8.0
```
Comment (kingthorin) [inline src/main/java/net/datafaker/providers/videogame/RedDeadRedemption2.java:8]: Can’t be. 1.8.1 is already released. Upcoming is 2.0.0

Comment (Majid-J) [reply src/main/java/net/datafaker/providers/videogame/RedDeadRedemption2.java:8]: Ofc! 😅

**Review by kingthorin — APPROVED** [review]
LGTM

File: src/main/java/net/datafaker/providers/videogame/RedDeadRedemption2.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
+package net.datafaker.providers.videogame;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+/**
+ * Red Dead Redemption 2 is an action-adventure game developed and published by Rockstar Games.
```
Comment (bodiam) [inline src/main/java/net/datafaker/providers/videogame/RedDeadRedemption2.java:6]: Thanks for adding this documentation here!!

**Comment by bodiam** [comment]
Thanks for this great PR! Looks good to me too!

File: src/test/java/net/datafaker/providers/videogame/RedDeadRedemption2Test.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        return List.of(
+            TestSpec.of(redDeadRedemption2::protagonist, "red_dead_redemption2.protagonists"),
+            TestSpec.of(redDeadRedemption2::gangMember, "red_dead_redemption2.gang_members"),
+            TestSpec.of(redDeadRedemption2::majorCharacter, "red_dead_redemption2.major_characters"),
+            TestSpec.of(redDeadRedemption2::animal, "red_dead_redemption2.animals"),
+            TestSpec.of(redDeadRedemption2::state, "red_dead_redemption2.states"),
+            TestSpec.of(redDeadRedemption2::region, "red_dead_redemption2.regions"),
+            TestSpec.of(redDeadRedemption2::settlement, "red_dead_redemption2.settlements"),
+            TestSpec.of(redDeadRedemption2::quote, "red_dead_redemption2.quotes"),
+            TestSpec.of(redDeadRedemption2::weapon, "red_dead_redemption2.weapons")
```
Comment (snuyanzin) [inline src/test/java/net/datafaker/providers/videogame/RedDeadRedemption2Test.java:21]: would be better if it satisfies abc order

Comment (snuyanzin) [inline src/test/java/net/datafaker/providers/videogame/RedDeadRedemption2Test.java:21]: would be better if it satisfies abc order

Comment (bodiam) [reply src/test/java/net/datafaker/providers/videogame/RedDeadRedemption2Test.java:21]: So picky :)

Comment (snuyanzin) [reply src/test/java/net/datafaker/providers/videogame/RedDeadRedemption2Test.java:21]: probably that was the reason github has rejected with submission of my first comment...
ok,,

**Pull request merged**